design/features/confirmation-email-for-form-fillers/version-1.md

```diff
@@ -0,0 +1,217 @@
+- contact information that the form creator has added to support their users  
+  
+![Access to work plus referral confirmation email example. Screenshot](https://github.com/alphagov/forms/assets/35372982/cf4d3bfd-b0d5-41b8-a8fc-281431861eee)  
+*This email shows an outline of the content and format that a typical email confirming that a submission's been sent might look like. This example shows information DWP might use for one of their forms.*
+
+![Apply for a county parish holding confirmation email example. Screenshot](https://github.com/alphagov/forms/assets/35372982/a884ec01-d0c8-4509-a6d1-09881530adf0)  
+*This email shows an outline of the content and format that a typical email confirming that a submission's been sent might look like. This example shows information Defra might use for one of their forms.*
+
+<br>
+
+## Design for form creators
+
+Giving form fillers control over whether or not they receive an email confirmation meant we needed to tell form creators about any information they provide which will be included in this email - and to make sure we do this at the appropriate stage of their form-creation journey. We decided that this would be at the point form creators add or edit their content for the "Information about what happens next" and "Contact details for support" pages. 
+
+### Form submitted page
+
+![Form submitted page. Screenshot](https://github.com/alphagov/forms/assets/35372982/24eabfbc-f961-43ea-8b4a-cc764af65a11)  
+
+We’ve added new content to this screen, just above the textarea input:
+
+> This content will also be included in an email confirming that a form's been submitted - if people choose to receive this. The email will include the date and time of submission, and contact details for support. It will not contain a copy of someone's answers.
+
+### Provide contact details for support
+
+![Provide contact details for support. Screenshot](https://github.com/alphagov/forms/assets/35372982/06fa052d-2cfd-4d31-aaea-1081ce02e187)  
+
+We’ve added new content to this screen, just above the “How can people get help with filling in this form?” question:
+
+> It will also be included in an email confirming that a form's been submitted - if people choose to receive this. The email will include the date and time of submission. It will not contain a copy of someone's answers.
+
+
+<br>
+
+## Notes
+
+We know that we’ll have to return to this feature later to include the following:  
+
+- Controls for form creators:  
+  - Allow them to dictate if the WHN information is or is not included in the confirmation email  
+  - Give them a way to make confirmation emails mandatory for all users  
+  - Explore if we should allow them to decide their own subject line for the emails  
+  - Explore if we should let them control whether submission data is or is not included in a confirmation email
+    - Does the submission copy need to act as a legal proof (due to policy. This will dictate file types such as PDF over .txt)
```

Comment:
So I think it should read as follows, but not sure if you or I should make this edit (in terms of getting into an endless 'review' loop!)?

- Does the submission copy need to act as a legal proof (due to policy)? This will dictate file types, such as PDF rather than .txt

Reply:
I've just made this edit (I think!)